scripts/cms5/phpstan.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+// Only valid for non-theme modules
+if (!is_module() || is_theme()) {
+    return;
+}
+
+// Get the dirs we want to run static analysis against
+$dirs = [];
+foreach (['code', 'src', 'app/src'] as $codeDir) {
+    if (check_dir_exists($codeDir)) {
+        $dirs[] = $codeDir;
+    }
+}
+
+// If we have some dirs to run against, we don't need to do anything.
+if (empty($dirs)) {
+    return;
+}
+
+$content = <<<EOT
+parameters:
+  paths:
+    - %s
+
+EOT;
+
+// Create a phpstan config file
+write_file_even_if_exists('phpstan.neon.dist', sprintf($content, implode("\n    - ", $dirs)));
```

Comment:
`.dist` is a standard for these. If we one day all agree to not use `.dist` files it will be easy to update this to remove the `.neon.dist` file and add the `.neon` one, but for now we should just do the standard thing.